src/client/components/leaderboard/leaderboard.js

```diff
@@ -18,6 +27,7 @@ class Leaderboard extends React.Component {
               <tr>
                 <th>#</th>
                 <th>Name</th>
+                <th></th>
```

Comment:
Would it be easier to give this column the heading "Passed" and just add a check mark if the user did pass?